index.js

```diff
@@ -249,6 +249,19 @@ function bootstrap(options) {
       return next();
     });
   }
+
```

Comment:
```suggestion
/**
 * Handles requests to the session timeout page.
 * - If the user has a session cookie but their session is missing or inactive,
 *   this triggers a session timeout error handled by error middleware.
 * - Otherwise, responds with a 404 "Page Not Found" error.
 * This route ensures the timeout page only appears after an actual session expiry.
 */
```